CMakeLists.txt

```diff
@@ -34,13 +34,13 @@ if (WIN32)
 endif ()
 
 if (MSVC)
-    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} /bigobj /wd4250 /wd4244")
+    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} /bigobj /wd4250 /wd4244 /Zc:preprocessor")
     set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} /bigobj")
     add_definitions(-D_CRT_SECURE_NO_WARNINGS)
-endif ()
-
-set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wall -pedantic")
-#set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -g -fsanitize=address -fno-omit-frame-pointer")
+else()
+    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wall -pedantic")
+    #set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -g -fsanitize=address -fno-omit-frame-pointer")
```

Comment:
Why is this commented out?

Reply:
I have no idea why, it was commented on the original as well (https://github.com/DiscordPP/echo-bot/blob/44a736a24f57dbcaee70804d200275c04ebcd36e/CMakeLists.txt#L43)

Reply:
Oh you're right, I'm blind lol.